monkestation/code/modules/particles/byond_particles/particle/burst.dm

```diff
@@ -0,0 +1,26 @@
+/particles/dust
+	width = 124
+	height = 124
+	count = 256
+	spawning = 999999 //spawn all instantly
+	lifespan = 0.75 SECONDS
+	fade = 0.35 SECONDS
+	position = generator("box", list(-16, -16), list(16, 16), NORMAL_RAND)
+	velocity = generator("circle", -8, 8, NORMAL_RAND)
+	friction = 0.125
+	color = COLOR_WHITE
+
+/particles/debris
+	width = 124
+	height = 124
+	count = 16
+	spawning = 999999 //spawn all instantly
```

Comment:
May want a define for huge numbers like this, so it looks more readable.
SPAWN_ALL_PARTICLES_INSTANTLY or something